ml_prototype/post_training/scripts/generate_character_data.py

```
@@ -0,0 +1,322 @@
+    with open(output_jsonl_path, "a") as f:
+        for topic in tqdm.tqdm(topic_list):
+            f.write(json.dumps(topic) + "\n")
+
+    print("Saved topics")
+
+
+def generate_questions():
+    pass
+
+
+def generate_data(
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)